enterprise/server/cmd/executor/yaml_doc/BUILD

```diff
@@ -34,4 +34,8 @@ genrule(
         cat $(location :generate_yaml) >> "$@" &&
         echo "\\`\\`\\`" >> "$@"
     """,
+    visibility = [
+        "//enterprise:__subpackages__",
```

Comment:
Do you need this line with line 3?

Reply:
My reading of "Rule target visibility" here: https://bazel.build/concepts/visibility is that the visibility attribute overrides the default_visibility of the containing package. But in this case it's fine because nothing in //enterprise depends on this rule, so I'll go ahead and remove it.